### PR TITLE
ENH: Add Pickable segment display property to allow disable point pic…

### PR DESCRIPTION
**Pull request by cpinter**
…king

### CODE REVIEW

**Comment by cpinter** [comment]
https://github.com/Slicer/Slicer/assets/1325980/ac32aa1b-bda8-42aa-b04b-81bc15a52a6b

File: Libs/MRML/Core/vtkMRMLSegmentationDisplayNode.h [hunk 1]
```diff
@@ -58,6 +58,8 @@ class VTK_MRML_EXPORT vtkMRMLSegmentationDisplayNode : public vtkMRMLDisplayNode
     double Opacity3D{1.0};
     double Opacity2DFill{1.0}; // This one is used for labelmap volume related operations (color table, merged labelmap)
     double Opacity2DOutline{1.0};
+    /// Pickable by interactions
```
Comment (lassoan) [inline Libs/MRML/Core/vtkMRMLSegmentationDisplayNode.h:61]: This is not very clear. I've kind of almost already forgotten what exactly we wanted to use this for and how it is related to the `Selectable` node property, and this short comment did not help me to understand what this flag is for. Maybe also mention that it is only for 3D views.

**Review by lassoan — COMMENTED** [review]
This is very nice and clear. I only had a small comment on improving the documentation, because the subtle difference in behavior difference it may not be obvious (many developer may not even know what "picking" refers to).

**Comment by cpinter** [comment]
Thank you for the review! I added more information in the comment about the Pickable property. I hope this explains it well enough.

**Review by lassoan — APPROVED** [review]
Thank you, it looks good.

**Pull request merged**